pages/vi/profiles/joelschwartz.md

```diff
@@ -0,0 +1,10 @@
+
+1. My name is Joel Schwartz
+2. I am from Mansfield Ohio 
+3. my time zone is eastern standard time
+![adirondack mountains view](https://www.wildernessphotographs.com/images/xl/Boreas-Mt-Treetop-crop-of-Elk-Lake-High-Peaks-_1000px.jpg)
```

Comment:
Looks much better. Awesome job!
However, the image and order number are shifted. To be able to fix that, you would need to add an empty line above the image.

![Screen Shot 2019-03-25 at 11 17 14 AM](https://user-images.githubusercontent.com/35533801/54943956-ac841b80-4eef-11e9-8871-c3b63a9b64fd.png)